build-config/make/sysroot.make

```diff
@@ -52,6 +55,10 @@ $(DEV_SYSROOT_INIT_STAMP): $(TREE_STAMP) | $(XTOOLS_BUILD_STAMP)
 	$(Q) echo "==== Preparing a new development sysroot ===="
 	$(Q) rm -rf $(DEV_SYSROOT)
 	$(Q) cp -a $$($(CROSSBIN)/$(CROSSPREFIX)gcc -print-sysroot) $(DEV_SYSROOT)
+	$(Q) rsync -rl $(XTOOLS_INSTALL_DIR)/$(TARGET)/$(TARGET)/lib $(DEV_SYSROOT)
+       ifeq ($(ARCH),$(filter $(ARCH),arm64 x86_64))
+		$(Q) rsync -rl $(XTOOLS_INSTALL_DIR)/$(TARGET)/$(TARGET)/lib64 $(DEV_SYSROOT)
+       endif
```

Comment:
@cbrune The problem with accton_wedge_16x is coming due to machine specific busybox patches that are based on old busybox version. i have ported these patches to new version and submitted the patch. please check

i did not encounter any problem with accton_as5712_54x . i also checked nxp_p2020rdbpca. busybox is building fine in both machines